.gitlab/package_build/windows.yml

```diff
@@ -40,6 +40,7 @@
       -e AGENT_FLAVOR=${AGENT_FLAVOR}
       -e OMNIBUS_SOFTWARE_VERSION="${OMNIBUS_SOFTWARE_VERSION}"
       -e OMNIBUS_RUBY_VERSION="${OMNIBUS_RUBY_VERSION}"
+      -e PYTHONUTF8=1
```

Comment:
non-blocking, but an inline comment with some justification/explanation would save someone some head-scratching later.

Reply:
That's a fair point, but I can't do that without changing the YAML multiline marker from `>` to `|`, which would probably require changing every line to add newline escaping, so in this case (given the limited impact of this flag) I prefer to not complicate the diff.

For what it's worth, its purpose is not too hard to find (https://docs.python.org/3/using/cmdline.html#envvar-PYTHONUTF8): in this case the output of the `cryptography` build when it failed included fancy emojis that caused a big part of the error message to be lost because of encoding problems.

Reply:
sigh, ok.